_test/test_mpi_wrappers/test_exchange_CppMPI.m

```diff
@@ -66,108 +66,6 @@ function test_MessagesCppMPI_constructor(obj)
         %-----------------------------------------------------------------
         % CURRENTLY DISABLED
         %
```

Comment:
Do we need this comment anymore?

Reply:
that's placeholder left in process to see if anything still needs to be disabled. Apparently not. Removing